explorer/src/Pos/Explorer/Web/Api.hs

```diff
@@ -35,6 +36,10 @@ type PageNumber = Integer
 -- | Common prefix for all endpoints.
 type API = "api"
 
+type TotalAda = API
+    :> "totalada"
```

Comment:
I would prefer to have named something more in the spirit of REST recommendations. 
So something like `currencies/ada` or `currencies?name=ada` (this is what I would prefer) or `currencies/ada(s)/total/` is more appropriate.
For example, this is a decent guide for the names - http://www.vinaysahni.com/best-practices-for-a-pragmatic-restful-api